ontology/EBUCore/ebucore.owl

```diff
@@ -3521,13 +3473,15 @@ ec:costumeTexture rdf:type owl:DatatypeProperty ;
 ec:dateTime rdf:type owl:DatatypeProperty ,
                      owl:FunctionalProperty ;
             rdfs:domain ec:DateTimeObject ;
-            rdfs:range xsd:dateTime .
+            rdfs:range xsd:dateTime ;
+            dcterms:description "A string describing the time and date."@en ;
+            rdfs:label "Date time"@en .
 
 
 ###  http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#day
 ec:day rdf:type owl:DatatypeProperty ;
        rdfs:range xsd:integer ;
-       rdfs:label "day"@en .
+       rdfs:label "Day"@en .
```

Comment:
Documentation could be improved: is it the name of the weekday or the day number in that week or...?

Reply:
rdfs:range xsd:integer
Doesn't that mean there can only be a number?
Is there a rule, which day a week starts?

Reply:
I think the next #58 pull request will solve this question, then ec:day will be changed to time:day
The same will be the case for month and year.
And day are related to month and not week, like it is in ISO8601